mne/tests/test_docstring_parameters.py

```diff
@@ -14,6 +14,7 @@
 public_modules = [
     # the list of modules users need to access for all functionality
     'mne',
+    'mne.baseline',
```

Comment:
Pushed this to reflect adding it to `python_reference.rst`

Reply:
Thanks!